### ISSUE TITLE
UITabBarItem selectedImage

### ISSUE THREAD

**Issue by fbomb111** [Vaberer/Font-Awesome-Swift#42]
Using setFAIcon on a tabBarItem only sets the image, so the image goes missing when you select that tab item.  Adding selectedImage to the extension will fix the problem:

```
`public extension UITabBarItem {

    public func setFAIcon(icon: FAType) {
        FontLoader.loadFontIfNeeded()
        image = UIImage(icon: icon, size: CGSize(width: 30, height: 30))
        selectedImage = UIImage(icon: icon, size: CGSize(width: 30, height: 30))
    }
}`
```

**Comment by uknowmeright** [comment]
Thanks! exactly what I needed. This should be added to the main repo

**Comment by Vaberer** [comment]
Hello, implemented in  v1.5.3
Thanks
